.github/workflows/pr-test.yml

```
@@ -25,35 +35,10 @@ jobs:
 
     steps:
       - uses: actions/checkout@v3
```

This should resolve the merge conflicts

```suggestion
      - uses: actions/checkout@v4
```